src/section/shared_state.rs

```diff
@@ -173,15 +179,32 @@ impl SharedState {
         // FIXME: we should perform these checks before voting, but once the vote accumulates we
         // must obey it:
         if !self.our_prefix().matches(p2p_node.name()) {
-            trace!("not adding node {} - not matching our prefix", p2p_node);
+            trace!(
+                "not adding node {} - not matching our prefix",
+                p2p_node.name()
```

Comment:
Why this change?

Reply:
just to make the log message cleaner and consistent with outputs in other places

Reply:
OK. I think it might sometimes be useful to see the full `P2pNode` in the log, not just the name, but it's probably fine here.